libnavigation-core/src/main/java/com/mapbox/navigation/core/routerefresh/RouteRefreshController.kt

```diff
@@ -118,6 +124,20 @@ internal class RouteRefreshController(
             )
             currentRequestId = null
         }
+
+        private fun getUpdatedLegs(oldLegs: List<RouteLeg>?, newLegs: List<RouteLeg>?): List<Int> {
+            val updatedLegs = arrayListOf<Int>()
+            if (oldLegs != null && newLegs != null) {
+                for (legIndex in currentLegIndex until min(oldLegs.size, newLegs.size)) {
+                    val oldLegCongestion = oldLegs[legIndex].annotation()?.congestion()
```

Comment:
Congestion is only one of the annotations, actually, any of them could've been refreshed.

Could we log a map of `leg index to annotation type updated` or something similar?